stories/topics/tech-note-aws-add-remove-tags.adoc

```diff
@@ -0,0 +1,7 @@
+[id="tech-note-aws-add-remove-tags"]
+
+= Add and Remove Tags
+
+{AAPonAWS} supports stack-level tagging during stack deployment time. A stack-level tag is applied to the stack itself and all tag supported components that make up the stack.
```

Comment:
Perhaps you might say what a tag is?

Reply:
I think AWS users would be familiar with tags, but if you do need to add some text about it, here is one example AWS has:
<img width="886" alt="image" src="https://github.com/ansible/aap-docs/assets/32707829/8bc3e053-04ee-4ce5-a17a-80cde9d3d756">

Reply:
Tags are custom fields that you assign to an AWS resource to help organize and identify your instances. {AAPonAWS} supports stack-level tagging during stack deployment time. A stack-level tag, when applied to the stack itself, is also applied to all tag supported components that make up the stack.